### ISSUE TITLE
Streaming a response

### ISSUE THREAD

**Issue by cyco130** [lagonapp/lagon#211]
**Describe the bug**
Returning a `Response` created from a `ReadableStream` doesn't seem to work.  

**To Reproduce**

1. Run the following file with `lagon dev stream.js`:

```
export function handler(request) {
  return new Response(
    new ReadableStream({
      pull(controller) {
        controller.enqueue(new Uint8Array([65, 66, 67]));
        controller.close();
      },
    }),
  );
}
```

2. Open `http://127.0.0.1:1234` in a browser.
3. Notice it hangs.

I also tried creating a `ReadableStream` via `TransformStream`.

**Expected behavior**
 It should display `ABC`.

**Information:**
 - OS: Linux Mint on amd64
 - Browser: Tried with a few (Chrome,  Firefox, curl)
 - Version: `lagon-cli 0.1.0`

**Comment by QuiiBz** [comment]
I *think* this has been fixed in #204, let me try. I haven't yet merged the release for this (https://github.com/lagonapp/lagon/pull/203).

**Comment by QuiiBz** [comment]
Can confirm that the issue was fixed by #204! I'll do a release soon, so you'll have to `npm i -g @lagon/cli` again to upgrade.

<img width="1546" alt="Screenshot 2022-10-25 at 09 14 41" src="https://user-images.githubusercontent.com/43268759/197707784-0f34f4b0-bbd5-4695-93a1-45f59b2b8a48.png">

**Comment by QuiiBz** [comment]
Just released version [0.3.3](https://github.com/lagonapp/lagon/releases/tag/%40lagon%2Fcli%400.3.3), which also includes some more fixes and improvements. Tested again with your code and it's working fine!

-> `npm i -g @lagon/cli@latest`
